shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +146,146 @@ shared_library("create_flutter_framework_dylib") {
   ]
 }
 
+ocmock_path = "../../../../../third_party/ocmock/Source"
+
+# TODO: Clone the OCMock repository so we can add a BUILD.gn to it.
```

Comment:
It already is cloned and patched I think. 

DEPS https://chromium.googlesource.com/chromium/src/+/master/third_party/ocmock/BUILD.gn and use like https://chromium.googlesource.com/chromium/src/+/master/ios/testing/BUILD.gn

Reply:
as commented above

Reply:
Added a todo issue.